tests/test_model.py

```diff
@@ -23,90 +23,90 @@ def grey_scale():
 
 
 def test_erode(cube, square):
-    assert np.sum(core.apply_basic_morpho(cube, 1, 'erosion', round_shape=False)) == 4**3
-    assert np.sum(core.apply_basic_morpho(cube, 2, 'erosion', round_shape=False)) == 2**3
+    assert np.sum(mdl().apply_basic_morpho(cube, 1, 'erosion', round_shape=False)) == 4**3
+    assert np.sum(mdl().apply_basic_morpho(cube, 2, 'erosion', round_shape=False)) == 2**3
```

Comment:
Instead of calling `mdl()` each time, is it not better to initiate the Model class in a variable and use it?